src/main/resources/templates/checkout.html

```
@@ -5,41 +5,116 @@
+
+    <div>
+      <label for="amount">amount</label>
+      <input
+              id="amount"
+              type="number"
+              placeholder="금액을 입력하세요"
+              style="display: block; margin: 8px 0 16px; padding: 8px; width: 300px;"
+      />
+    </div>
+
+    <button class="button" onclick="requestPayment()">결제하기</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)